packages/server-wallet/jest/chain-setup.ts

```diff
@@ -6,6 +6,10 @@ import {defaultConfig} from '../src/config';
 import {deploy} from '../deployment/deploy';
 
 export default async function setup(): Promise<void> {
+  process.env['GANACHE_HOST'] = '0.0.0.0';
+  process.env['GANACHE_PORT'] = '8545';
+  process.env['RPC_ENDPOINT'] = `http://${process.env['GANACHE_HOST']}:${process.env['GANACHE_PORT']}`;
```

Comment:
Why are these environment variables defined here, a reviewer such as yourself might ask.

The wallet instantiates a chain service if an rpc endpoint and a chain private key are defined. For non-chain unit tests, ganache is not running. So the chain service instantiation will fail if these environment variables are defined (since default wallet configuration is populated from environment variables).

Some alternative approaches are be:
- Use different env var files for tests vs chain tests.
- Add a configuration to the wallet that determines whether or not a chain service should be instantiated.